### ISSUE TITLE
Fix Request: false positive with AvoidModifyingObjectMapper when the objectMapper is being updated in the PostConstruct.

### ISSUE THREAD

**Issue by hem-verma** [jborgers/PMD-jPinpoint-rules#148]
Getting AvoidModifyingObjectMapper even when the objectMapper is being updated with in the PostConstruct. So the thread safety issue mentioned would not be encountered.

```
private static final ObjectMapper MAPPER;
    
static {
        MAPPER = new ObjectMapper();
    }

    @PostConstruct
    private synchronized void init() {
        staticPropositionProductDeserializer = this.propositionProductDeserializer;
        staticPropositionProductSerializer = this.propositionProductSerializer;
        //register Custom Deserializers
        SimpleModule moduleDeserializer = new SimpleModule();
        moduleDeserializer.addDeserializer(PropositionProduct.class, staticPropositionProductDeserializer);
        MAPPER.registerModule(new JodaModule()); 
        MAPPER.registerModule(moduleDeserializer); 
    }
```

**Comment by jborgers** [comment]
This is a duplicate of #116
